solutions/standard/variables.tf

```diff
@@ -54,64 +54,64 @@ variable "existing_secrets_endpoint_type" {
   default     = "private"
   validation {
     condition     = contains(["public", "private"], var.existing_secrets_endpoint_type)
-    error_message = "Allowed values for 'existing_secrets_endpoint_type' are \"public\" and \"private\"."
+    error_message = "Only \"public\" and \"private\" are allowed values for 'existing_secrets_endpoint_type'."
```

Comment:
We want to add the information to the description before they get the error (line 53)

description = "The endpoint type to use if `existing_secrets_manager_crn` is specified. Possible values: `public`, `private`."